### PR TITLE
Invalid file import

### LINKED ISSUE #67: Crashes when opening invalid file
At the moment PixiEditor crashes when trying to open a invalid/corrupted file

It would be good if it would display a error message instead

### PR DESCRIPTION
**Pull request by flabbet**
Fixes #67 

Changes: Added `CorruptedFileException` which is thrown by `ImportImage` and `ImportDocument`. Wrapped `Open()` method with try and catch, whenever `CorruptedFileException` is thrown, error MessageBox is shown to the user. Also changed framework version to .NET 5.

Example of usage: Just rename any non-image file to .jpg, .png, or .pixi extension and try to open it in PixiEditor.

Dependencies cleared: Yup

Tests passing: Yup

### CODE REVIEW

**Comment by flabbet** [comment]
@CPKreuz Do you mind reviewing?

**Comment by CPKreuz** [comment]
Will do tomorrow

File: PixiEditor/ViewModels/ImportFilePopupViewModel.cs [hunk 1]
```diff
@@ -132,12 +132,19 @@ private void CheckForPath(string path)
...
-                _filePath = path;
-                BitmapImage bitmap = new BitmapImage(new Uri(path));
-                ImportHeight = bitmap.PixelHeight;
-                ImportWidth = bitmap.PixelWidth;
+                try
+                {
+                    PathButtonBorder = "#b8f080";
+                    PathIsCorrect = true;
+                    filePath = path;
+                    BitmapImage bitmap = new BitmapImage(new Uri(path));
```
Comment (CPKreuz) [inline PixiEditor/ViewModels/ImportFilePopupViewModel.cs:140]: After hex editing a image file it threw an `FileFormatException` with the message 'The image format is unrecognized.'

**Review by CPKreuz — CHANGES_REQUESTED** [review]

**Review by CPKreuz — APPROVED** [review]
Seems to work now

**Pull request merged**